### ISSUE TITLE
Getting bad results When I ran drqn experiments .

### ISSUE THREAD

**Issue by Alex-zhai** [5vision/DARQN#2]
Hello, Ask you a question. When I ran exp11, the drqn agent seems not as good as dqn. I ran five games, and the drqn results are all lower than dqn. What caused this phenomenon.

**Comment by 5vision** [comment]
You are right. We are unable to reproduce exact results from the [paper](http://arxiv.org/abs/1507.06527) by Matthew et al. Probably, we should remove a fully connect layer before LSTM or use the same hyper-parameters, etc. We decided not to waste time on it. For all the results of DRQN in our [paper](https://arxiv.org/abs/1512.01693), we used the [implementation](https://github.com/mhauskn/dqn/tree/recurrent) on Caffe.

**Comment by Alex-zhai** [comment]
So, the implemention on Caffe has DRQN module?

**Comment by Alex-zhai** [comment]
Can you give me some instruction of installation of DRQN on caffe? Thank you!!!

**Comment by Alex-zhai** [comment]
And your environment of running DRQN on caffe is Ubantu??

**Comment by Alex-zhai** [comment]
One more question, when I ran Hard-Darqn, I get `cannot open <pretrain_seaquest.t7> in mode r  at /tmp/luarocks_torch-scm-1-5561/torch7/lib/TH/THDiskFile.c:484`. So where should I put the `pretrain_seaquest.t7` model.
